cpp/src/arrow/table.h

```diff
@@ -101,6 +101,19 @@ class ARROW_EXPORT Table {
   static Result<std::shared_ptr<Table>> FromChunkedStructArray(
       const std::shared_ptr<ChunkedArray>& array);
 
+  /// \brief Convert table with one data type to Tensor
```

Comment:
This "with one data type" is probably a left-over from the first version (when only that was supported)?

(same for the record batch doc)